galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.cloud.manager/src/main/java/dev/galasa/cloud/internal/properties/ContainerOverrideProvider.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright contributors to the Galasa project 
+ */
+package dev.galasa.cloud.internal.properties;
+
+import dev.galasa.cloud.CloudManagerException;
+import dev.galasa.framework.spi.cps.CpsProperties;
+
+
+/**
+ * Default provider of cloud containers
+ * 
+ * @galasa.cps.property
+ * 
+ * @galasa.name cloud.default.provider
+ * 
+ * @galasa.description Specifies the default provider of cloud containers
+ * 
+ * @galasa.required No
+ * 
+ * @galasa.default docker
+ * 
+ * @galasa.valid_values A valid provider name, depending on CPS configuration could be, docker, k8s, gcprun, awsfargate etc
+ * 
+ * @galasa.examples 
+ * <code>cloud.default.provider=k8s<br>
+ * </code>
+ * */
+public class ContainerOverrideProvider extends CpsProperties {
```

Comment:
Looks like the content of this class is the wrong way around with the default provider

Reply:
corrected